wiki/Tournaments/CWC/1/en.md

```diff
@@ -0,0 +1,262 @@
+---
+tags:
+- CWC
+---
+# Catch the Beat World Cup
+
+The **Catch the Beat World Cup** (***CWC*** ) was a team-based Catch the Beat tournament hosted by the [osu! staff](/wiki/People/The_Team). It was the 1st installment of the osu! Catch the Beat World Cup.
```

Comment:
`(***CWC*** )` -> `(***CWC***)`
`team-based Catch the Beat tournament` -> `team-based osu!catch tournament`
`osu! staff` -> `osu! team`
`installment of the osu! Catch the Beat World Cup` -> `installment of the osu!catch World Cup`

Check other articles.